src/datasets/table.py

```diff
@@ -1902,10 +1902,18 @@ def _concat_arrays(arrays):
                 _concat_arrays([array.values for array in arrays]),
             )
         elif pa.types.is_fixed_size_list(array_type):
-            return pa.FixedSizeListArray.from_arrays(
-                _concat_arrays([array.values for array in arrays]),
-                array_type.list_size,
-            )
+            if config.PYARROW_VERSION.major < 13:
+                # PyArrow bug: https://github.com/apache/arrow/issues/35360
```

Comment:
how does it relate to this issue about hashing ?

Reply:
[Here](https://github.com/apache/arrow/issues/35573#issuecomment-1547224289) it's mentioned that fixing this issue should also solve the issue with sliced fixed-size list arrays

Reply:
ah I see !